crux-core/src/crux/api/ICruxAPI.java

```diff
@@ -156,8 +156,8 @@
      */
     @SuppressWarnings("unchecked")
     public static final Map<Keyword, ?> TX_INDEXED_EVENT_OPTS = (Map<Keyword, Object>) PersistentArrayMap.EMPTY
-        .assoc(Keyword.intern("crux/event-type"), Keyword.intern("crux/tx-indexed"))
-        .assoc(Keyword.intern("with-tx-ops?"), true);
+            .assoc(Keyword.intern("crux/event-type"), Keyword.intern("crux/tx-indexed"))
```

Comment:
Could you exclude this indentation change from the PR if it's not related?

Reply:
Pants, I think intellij automatically redid the the indentation when I touched various things. 
I won't reply to all of the examples, but I'll sort out the indentation in all instances